src/appmain.js

```diff
@@ -116,133 +117,144 @@ class App extends LitElement
     }
     return html`
     ${!this.embed || this.embed === "full" ? html`
-      <nav class="navbar has-background-info" role="navigation" aria-label="main navigation">
+    <a href="#skip-main-target" @click=${this.skipMenu} class="skip-link">Skip main navigation</a>
+    <nav class="navbar has-background-info" aria-label="main">
       <div class="navbar-brand">
-        ${!this.embed ? html`
-        <a class="navbar-item wr-logo-item" title="ReplayWeb.page" href="/">
-          <fa-icon id="wrlogo" size="2.5rem" .svg=${rwpLogo}></fa-icon>
-        </a>` : html`
-        <a class="navbar-item wr-logo-item" title="ReplayWeb.page">
-          <fa-icon id="wrlogo" size="2.5rem" .svg=${rwpLogo}></fa-icon>
-        </a>
+        ${!this.embed ? html `
+          <a href="/" class="navbar-item wr-logo-item menu-only" aria-labelledby="home">
+            <fa-icon id="wrlogo" size="2.5rem" .svg=${rwpLogo} aria-hidden="true"></fa-icon>
+          </a>
+          <span class="navbar-item wr-logo-item wide-only">
+            <fa-icon id="wrlogo" size="2.5rem" .svg=${rwpLogo} aria-hidden="true"></fa-icon>
+          </span>`: html `
+          <span class="navbar-item wr-logo-item">
+            <fa-icon id="wrlogo" size="2.5rem" .svg=${rwpLogo} aria-hidden="true"></fa-icon>
+          </span>
         `}
```

Comment:
Seems like this was maybe a merge error, the wrlogo is repeated 3 times, twice in the first option..

Reply:
This is not a merge error: this is actually addressing the intricacy of the three different situations the logo finds itself, in your present design.

If `!this.embed`:
- On big screens (`class="wide-only"`), you have the logo, kept in the `navbar-brand` container, visually abutting the ReplayWeb.page wordmark (`  ${!this.embed ? html <a id="home" class="navbar-item logo-text`), the first element in the collapsible nav menu.
- On small screens (`class="menu-only"`), the logo stands alone, and the wordmark appears to the mobile popup menu.
If `this.embed`,
-  the logo stands alone at all screen sizes, but, previous to this PR, was set to `<a class="navbar-item wr-logo-item" title="ReplayWeb.page">`, an invalid `<a>`, since it lacks an `href`.

With the proposed markup:
- the logo still only appears once
- the duplicative link to `/` is removed when the wordmark is present, so that visitors don't have to navigate through a duplicative link immediately, at the top of every page
- when the logo does not link to anywhere, it is a `<span>` rather than an `<a>`

If you want users to be able to click on both the logo and the wordmark on desktop in non-embed mode to navigate to `/`, that needs to be in a single `<a>`.  E.g., you need a wordmark in the `navbar-brand` container that disappears on mobile, and a `menu-only` wordmark inside the collapsible menu. 

I'll push that alternative markup up; either are equally valid HTML and are equally accessible.